File: src/pocketcloud/cloud/event/impl/player/PlayerEvent.php
<?php

namespace pocketcloud\cloud\event\impl\player;

use pocketcloud\cloud\event\Event;
use pocketcloud\cloud\player\CloudPlayer;

abstract class PlayerEvent extends Event {

    public function __construct(private readonly CloudPlayer $player) {}

    public function getPlayer(): CloudPlayer {
        return $this->player;
    }
}

File: cloud-core/src/main/java/de/pocketcloud/cloud/event/impl/player/PlayerEvent.java
package de.pocketcloud.cloud.event.impl.player;

import de.pocketcloud.cloud.event.CloudEvent;
import de.pocketcloud.cloud.player.CloudPlayer;
import lombok.Getter;

public abstract class PlayerEvent extends CloudEvent {

    @Getter
    private final CloudPlayer player;

    public PlayerEvent(CloudPlayer player) {
        this.player = player;
    }
}